lightning-block-sync/src/poll.rs

```diff
@@ -175,7 +175,7 @@ impl<B: DerefMut<Target=T> + Sized + Sync + Send, T: BlockSource> ChainPoller<B,
 	}
 }
 
-impl<B: DerefMut<Target=T> + Sized + Sync + Send, T: BlockSource> Poll for ChainPoller<B, T> {
+impl<B: DerefMut<Target=T> + Sized + Send + Sync, T: BlockSource> Poll for ChainPoller<B, T> {
```

Comment:
nit: excess diff.

Reply:
wasn't on purpose, but now I notice that `Send + Sync` is much more prevalent in the code - in fact there is only one instance of the reverse left after this PR